aqua-ethos-setup/aqua_ethos.sh

```diff
@@ -37,6 +37,30 @@ if [[ -z $TOKEN ]]; then
 	exit 1
 fi
 
+# Add the secomp profile if it doesnt exist
+read -r -d '' _SECCOMP_PROFILE <<'EOF'
+{"defaultAction":"SCMP_ACT_ERRNO","syscalls":[{"names":["capget","capset","chdir","fchown","futex","getdents64","getpid","getppid","lstat","openat","prctl","setgid","setgroups","setuid","stat","rt_sigaction","mprotect","brk","close","open","write","mmap","rt_sigprocmask","sched_getaffinity","arch_prctl","access","getrandom","sigaltstack","getrlimit","set_tid_address","fstat","stat","setsockopt","read","openat","clone","set_robust_list","ioctl","execve","gettid","socket","munmap","futex","bind"],"action":"SCMP_ACT_ALLOW","args":[],"comment":"Necessary syscalls for working of container","includes":{},"excludes":{}}]}
```

Comment:
Sorry to be nitpicky on this, but can you use the same spacing/line breaks as you do below? It makes it very hard to read/update in the future as one big line.